Compile failed on Xcode 10

```
$ make GCC=/usr/local/Cellar/gcc/8.3.0_2/bin/gcc-8
...
...
...
/usr/local/Cellar/gcc/8.3.0_2/bin/gcc-8 -m32 -Wl,-s -o gocpp0  ../drv_stdc/cpp0main.obj cppmain.o intl.o \
		libcpp.lib hashtable.o version.o ../intl/libintl.lib ../libiberty/libiberty.lib ../go_lib/go_lib.lib ../go_lib/stdin.o ../drv_stdc/drv_stdc.obj
ld: warning: option -s is obsolete and being ignored
ld: warning: The i386 architecture is deprecated for macOS (remove from the Xcode build setting: ARCHS)
ld: warning: ignoring file /usr/local/Cellar/gcc/8.3.0_2/lib/gcc/8/libgcc_ext.10.5.dylib, missing required architecture i386 in file /usr/local/Cellar/gcc/8.3.0_2/lib/gcc/8/libgcc_ext.10.5.dylib (1 slices)
ld: warning: ignoring file /usr/local/Cellar/gcc/8.3.0_2/lib/gcc/8/gcc/x86_64-apple-darwin18/8.3.0/libgcc.a, file was built for archive which is not the architecture being linked (i386): /usr/local/Cellar/gcc/8.3.0_2/lib/gcc/8/gcc/x86_64-apple-darwin18/8.3.0/libgcc.a
ld: warning: ignoring file /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib/libSystem.tbd, missing required architecture i386 in file /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib/libSystem.tbd
...
...
...
ld: symbol(s) not found for architecture i386
collect2: error: ld returned 1 exit status
make[3]: *** [gocpp0] Error 1
make[2]: *** [all] Error 2
make[1]: *** [all] Error 2
make: *** [all] Error 2
```

## Environments

- macOS : `10.14.5 (mojave)`
- Macbook Pro 13 inch
- Xcode : `10.2.1`
- `gcc` installed by Homebrew

--- a/README.md
+++ b/README.md
@@ -40,3 +40,16 @@ Source set of tools for developing Haribote OS .
 ### `sys/cdefs.h: No such file or directory`とのエラーが出る
 - Ubuntuの場合は、以下のコマンドを実行して、必要なパッケージをインストールしてください。(Thanks @sk2sat!)
  - `apt-get install libc6-dev-i386`
+
+### Xcode 10 以降でコンパイルできない
+
+macOS でコンパイルする場合 Xcode 10 以降は `i386` アーキテクチャは [サポートされなくなる](https://developer.apple.com/documentation/xcode_release_notes/xcode_10_release_notes#3035631) ためコンパイルができなくなります。
+
+以下の手順で Xcode 9.4 以前をダウンロードし、内部にある `MacOSX.sdk` を利用してコンパイルしてください。
+
+- [https://developer.apple.com/download/more/](https://developer.apple.com/download/more/) から `Xcode_9.4.1.xip` をダウンロードして解凍
+- `gcc` の `--sysroot` にダウンロードした SDK を設定してコンパイルする
+  - (解凍した `Xcode.app` を `/Applications/Xcode_9.4.1.app` に移動させた場合)
+  ```bash
+  make GCC="/usr/local/Cellar/gcc/6.2.0/bin/gcc-6 --sysroot=/Applications/Xcode_9.4.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk/"
+  ```

